src/controllers/employee.js

```
@@ -1,180 +1,606 @@
+      data.color = predefinedColors[i];
     }
   }
-  for (let emp of teamLeadId) {
-    if (emp.id && !ids.includes(emp.id)) {
-      ids.push(emp.id);
+  return arr;
+};
+
+//Utility function to change photo to base64
+const convertPhotoToBase64 = (employee) => {
+  employee.photo = employee.photo && employee.photo.toString("base64");
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)